SU2_CFD/include/output/COutput.hpp

```diff
@@ -0,0 +1,774 @@
+    short  offset; 
+    /*! \brief The group this field belongs to. */
+    string outputGroup;
+    /*! \brief String containing the description of the field */
+    string description;
+    /*! \brief Default constructor. */
+    VolumeOutputField () {}
+    /*! \brief Constructor to initialize all members. */    
+    VolumeOutputField(string fieldName_, int offset_, string volumeOutputGroup_, string description_):
+      fieldName(std::move(fieldName_)), offset(std::move(offset_)), 
+      outputGroup(std::move(volumeOutputGroup_)), description(std::move(description_)){}
+  };
+  
+  /*! \brief Associative map to access data stored in the volume output fields by a string identifier. */  
+  std::map<string, VolumeOutputField >          volumeOutput_Map;
+  /*! \brief Vector that contains the keys of the ::volumeOutput_Map in the order of their insertion. */  
+  std::vector<string>                           volumeOutput_List;
+  
+  /*! \brief Vector to cache the positions of the field in the data array */
+  std::vector<short>                            fieldIndexCache;
+  /*! \brief Current value of the cache index */  
+  unsigned short                                cachePosition;
+  /*! \brief Boolean to store whether the field index cache should be build. */    
+  bool                                          buildFieldIndexCache;
+  /*! \brief Vector to cache the positions of the field in the data array */
+  std::vector<short>                            fieldGetIndexCache;
+  /*! \brief Current value of the cache index */  
+  unsigned short                                curGetFieldIndex;
+
+  /*! \brief Requested volume field names in the config file. */    
+  std::vector<string> requestedVolumeFields;
+  /*! \brief Number of requested volume field names in the config file. */      
+  unsigned short nRequestedVolumeFields;
+    
+  /*----------------------------- Convergence monitoring ----------------------------*/     
+
+  su2double cauchyValue,         /*!< \brief Summed value of the convergence indicator. */
+  cauchyFunc;                    /*!< \brief Current value of the convergence indicator at one iteration. */
+  unsigned short Cauchy_Counter; /*!< \brief Number of elements of the Cauchy serial. */
+  su2double *cauchySerie;        /*!< \brief Complete Cauchy serial. */
+  su2double oldFunc,             /*!< \brief Old value of the coefficient. */
+  newFunc;                       /*!< \brief Current value of the coefficient. */
+  bool convergence;              /*!< \brief To indicate if the solver  has converged or not. */
+  su2double initResidual;        /*!< \brief Initial value of the residual to evaluate the convergence level. */
+  string convField;              /*!< \brief Name of the field to be monitored for convergence */
+  
+  /*----------------------------- Adaptive CFL ----------------------------*/     
+  
+  su2double rhoResNew,    /*!< New value of the residual for adaptive CFL routine */
```

Comment:
Adaptive CFL in the output can be removed. It is replaced with a new version located in the solver in #790

Reply:
Let me check whether there are any regression tests that use adaptive CFL. If not I am gonna remove it.

Reply:
There a couple of cases that use adaptive CFL. So lets remove it in the other PR then.